src/components/markdown-renderers/CodeBlock.js

```diff
@@ -19,7 +19,11 @@ export default class CodeBlock extends React.PureComponent {
     return (
       <SyntaxHighlighter
         style={tomorrowNightEighties}
-        customStyle={{ borderRadius: '2px', textAlign: 'left' }}
+        customStyle={{
+          borderRadius: '0.25em',
+          textAlign: 'left',
+          padding: '0.3em 0.3em calc(0.3em + 1px)',
```

Comment:
The `calc(0.3em + 1px)` is so that the content looks visually centered. Brains are weird and vertically centered things look like they're too far down. This bumps the bottom padding by a pixel, which makes the code blocks look more centered.